ansible/vars/aeternity/uat_mon.yml

```diff
@@ -0,0 +1,59 @@
+public_ipv4: "{{ ansible_ec2_public_ipv4|default(ansible_ssh_host)|default(ansible_host)|default(inventory_hostname) }}"
+datadog_enabled: yes
+api_base_uri: http://{{ public_ipv4 }}:{{ node_config.http.external.port }}/v2
+configure_peers: false
+seed_peers:
+  - "52.10.46.160"
+  - "18.195.109.60"
+  - "13.250.162.250"
+  - "13.53.161.215"
+
+node_config:
+  peers:
```

Comment:
UAT seed peers is automatically configured based on network_id since https://github.com/aeternity/aeternity/commit/b2454cdd0064264f47a09dab89dde188367f4709

Since we're going to deploy latest here anyway, we don't need this.